google-cloud-memcache-bom/pom.xml

```diff
@@ -39,6 +39,20 @@
     <url>https://github.com/googleapis/java-memcache</url>
   </scm>
 
+  <profiles>
```

Comment:
Does this need to be in a profile? Can we just always set `maven.antrun.skip` to true?

Reply:
It gets executed on the first execution, it probably doesn't build anything, but I was trying to minimize changes to the existing code.

Reply:
done